lib/legacy_activity.rb

```diff
@@ -0,0 +1,17 @@
+# require "csv"
+
+class LegacyActivity
+  attr_accessor :activity_node_set
+  def initialize(activity_node_set:)
+    self.activity_node_set = activity_node_set
+  end
+
+  def elements
+    activity_node_set.elements
+  end
+
+  def to_xml
+    activity_node_set.to_xml
+  end
+  end
```

Comment:
You have an extra `end` here but it's removed in a later commit!

Reply:
Fixed 👍